cloud_service/installing_cloud_other/prerequisites-cloud-other.adoc

```diff
@@ -11,9 +11,6 @@ toc::[]
 
 You must complete prerequisites before installing {product-title-managed} for supported Kubernetes platforms.
```

Comment:
```suggestion
You must complete prerequisites before installing {product-title-managed} for the supported Kubernetes platforms.
```

Reply:
I'm using no article to indicate "some" Kubernetes platforms (rather than all). IBM Style Guide refers to [this resource](https://gallaudet.edu/student-success/tutorial-center/english-center/grammar-and-vocabulary/definite-and-indefinite-articles/when-to-use-a-an-or-the/) when talking about articles. I feel this falls into the guidance in the table for _Don’t Use “a,” “an,” or “the”_?